tasks/concurrent.js

```diff
@@ -49,8 +53,21 @@ module.exports = function (grunt) {
 			});
 
 			if (opts.logConcurrentOutput) {
-				cp.stdout.pipe(padStream(' ', 4)).pipe(process.stdout);
-				cp.stderr.pipe(padStream(' ', 4)).pipe(process.stderr);
+				var taskLabel,numSpaces;
+				if(opts.logTaskName)
+				{
+					var colorFn = colorsAvailable.length > 0 ? colors[colorsAvailable.shift()] : function(s) { return s };//use an available color or none if more tasks then colors available
+					var maxLength = typeof opts.logTaskName === 'number' ?  opts.logTaskName : Math.min(maxTaskLength,12);//task labels in output up to 12 characters
+					taskLabel = '['+colorFn(task.substr(0,maxLength))+']';
```

Comment:
Use `slice` instead of `substr`.

Explanation: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/65